### PR TITLE
feat: add design embeds

### PR DESCRIPTION
**Pull request by 35C4n0r**
**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit aeff1c4792650e1d74cdb1dd4014052ad1927e90 at: https://655b7e81ddbcd028d2210de2--layer5.netlify.app

**Comment by leecalcote** [comment]
This console error is also present on the current production site:

```
Error: Invalid value for <svg> attribute width="auto"
```

File: src/sections/Home/Design-Embedds/index.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+              <h2>
+                <span className="embed-highlight">Showcase</span> your Designs
+              </h2>
+            </SectionTitle>
+            {/* </div> */}
+          </Col>
+          {/* <div className="bb"> */}
+          {/* </div> */}
+          <Col>
+            <div id="embedded-design-168bf17a-9386-4a0e-8c32-dd2bc22c8fe9">
```
Comment (aabidsofi19) [inline src/sections/Home/Design-Embedds/index.js:33]:
```suggestion
            <div id="embedded-design-168bf17a-9386-4a0e-8c32-dd2bc22c8fe9" style="height:26rem;width:100%">
```

**Comment by l5io** [comment]
🚀 Preview for commit 91b887ee0a4b25c139d3312935f01b4cbc8f41d9 at: https://655ca3c85df8ab7f8751e998--layer5.netlify.app

**Comment by ritiksaxena124** [comment]
I am getting this error in the console in the preview build @35C4n0r
image

File: src/sections/Home/Design-Embedds/index.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+              <h2>
+                <span className="embed-highlight">Showcase</span> your Designs
+              </h2>
+            </SectionTitle>
+            {/* </div> */}
+          </Col>
+          {/* <div className="bb"> */}
+          {/* </div> */}
+          <Col>
+            <div id="embedded-design-168bf17a-9386-4a0e-8c32-dd2bc22c8fe9" style="height:26rem;width:100%">
```
Comment (aabidsofi19) [inline src/sections/Home/Design-Embedds/index.js:33]:
```suggestion
            <div id="embedded-design-168bf17a-9386-4a0e-8c32-dd2bc22c8fe9" >
```

**Comment by 35C4n0r** [comment]
> I am getting this error in the console in the preview build @35C4n0r image

That was cause by this commit https://github.com/layer5io/layer5/pull/5140/commits/91b887ee0a4b25c139d3312935f01b4cbc8f41d9

It has ben reverted now.

**Comment by ritiksaxena124** [comment]
SVG or the image is not visible 
image

**Comment by l5io** [comment]
🚀 Preview for commit c169502f1a8aec647b34cab2e967112eadf0fae2 at: https://655d7bd493087546d83f79f1--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit c7a9d3f05bb4e4b411e1589856558272192d2c86 at: https://65633b02266830686f2cf1bc--layer5.netlify.app

**Pull request closed without merging**